src/library/features/crates/cratestorage.h

```diff
@@ -215,17 +46,6 @@ class CrateStorage: public virtual /*implements*/ SqlStorage {
     bool onDeletingCrate(
             CrateId crateId);
 
-    bool onAddingCrateTracks(
-            CrateId crateId,
-            const QList<TrackId>& trackIds);
-
-    bool onRemovingCrateTracks(
-            CrateId crateId,
-            const QList<TrackId>& trackIds);
-
-    bool onPurgingTracks(
-            const QList<TrackId>& trackIds);
-
 
     /////////////////////////////////////////////////////////////////////////
     // Crate read operations (read-only, const)
```

Comment:
It is not immediately obvious what it means to "read" a crate. It seems to mean updating the attributes of a `Crate` object with information in the database. That should be documented here.